<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWidget</class>
 <widget class="QWidget" name="CMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PurrFX procedural audio</string>
  </property>
  <layout class="QVBoxLayout" name="layMain">
   <item>
    <layout class="QGridLayout" name="layOptions">
     <item row="7" column="1">
      <widget class="QComboBox" name="cbxRegister4006Mode"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblRegister4001">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4001:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="cbxRegister400FMode"/>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="spbRegister4007Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="spbRegister4008Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spbRegister4004Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbxRegister4000Mode"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbxRegister4001Mode"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblRegister4004">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4004:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbxRegister4002Mode"/>
     </item>
     <item row="11" column="2">
      <widget class="QSpinBox" name="spbRegister400BValue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblRegister4003">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4003:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="cbxRegister400CMode"/>
     </item>
     <item row="11" column="1">
      <widget class="QComboBox" name="cbxRegister400BMode"/>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="cbxRegister400AMode"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lblRegister400C">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$400C:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lblRegister400B">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$400B:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QSpinBox" name="spbRegister400FValue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblRegister400A">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$400A:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblRegister4006">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4006:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spbRegister4002Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spbRegister4001Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblRegister4000">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4000:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="lblRegister400F">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$400F:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QSpinBox" name="spbRegister4005Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QSpinBox" name="spbRegister400CValue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="cbxRegister4008Mode"/>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="cbxRegister4007Mode"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbxRegister4004Mode"/>
     </item>
     <item row="7" column="2">
      <widget class="QSpinBox" name="spbRegister4006Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="cbxRegister400EMode"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spbRegister4000Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblRegister4008">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4008:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblRegister4005">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4005:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblRegister4002">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4002:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cbxRegister4005Mode"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lblRegister400E">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$400E:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblRegister4007">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>$4007:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QSpinBox" name="spbRegister400AValue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QSpinBox" name="spbRegister400EValue">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spbRegister4003Value">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="displayIntegerBase">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cbxRegister4003Mode"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layQuickSetup">
     <item>
      <widget class="QPushButton" name="btnNone">
       <property name="text">
        <string>Set all to &quot;None&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layControl">
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
